hardware/ariadne/bootloaders/ariadne/tftp.c

```diff
@@ -37,7 +37,13 @@ const unsigned char tftp_unknown_error_packet[] PROGMEM = "\0\5" "\0\0" "Error";
 #define TFTP_INVALID_IMAGE_LEN 23
 const unsigned char tftp_invalid_image_packet[] PROGMEM = "\0\5" "\0\0" "Invalid image file";
 
-uint16_t lastPacket = 0, highPacket = 0;
+uint8_t tftpFlashing;
+
+#ifndef TFTP_RANDOM_PORT
+static uint16_t tftpTransferPort;
+#endif
+
+static uint16_t lastPacket = 0, highPacket = 0;
```

Comment:
I am not confusing it with the meaning static has when declaring functions because as you can see in the source many functions are declared as static, which, in fact saves space in the binary. For that much I am sure.

But you are right that static has no effect in size when used with global variables to define scope, so no harm done, i guess.